rollup/internal/controller/sender/estimategas.go

```diff
@@ -57,7 +63,7 @@ func (s *Sender) estimateDynamicGas(auth *bind.TransactOpts, contract *common.Ad
 	}, nil
 }
 
-func (s *Sender) estimateGasLimit(opts *bind.TransactOpts, contract *common.Address, input []byte, gasPrice, gasTipCap, gasFeeCap, value *big.Int, minGasLimit uint64) (uint64, error) {
+func (s *Sender) estimateGasLimit(opts *bind.TransactOpts, contract *common.Address, input []byte, gasPrice, gasTipCap, gasFeeCap, value *big.Int) (uint64, error) {
```

Comment:
How about add retry to `s.client.EstimateGas`?

Reply:
I think it's quite uncommon that `s.client.EstimateGas` fails once and succeeds immediately afterwards. Usually it would fail when:
- There is a connection issue,
- The previous commit batch tx is not executed yet.

Reply:
Yeah. we can consider adding retry to prevent network instability, but it can be naturally retried in next loop of `gas-oracle` or `rollup-relayer`.

and if it's due to contract call failure in `EstimateGas` (e.g., parent mismatch), we should wait for the parent batch to be committed.